terratest/src/test/pega/data/values_with_overidden_liveness_probe_config.yaml

```diff
@@ -0,0 +1,375 @@
+          # pass the certificateChainFile file if you are using certificateFile and certificateKeyFile
+          cacertificate:
+          # provide the SSL certificate and private key as a PEM format
+          certificateFile:
+          certificateKeyFile:
+          # if you will deploy traefik addon chart and enable traefik, set enabled=true; otherwise leave the default setting.
+          traefik:
+            enabled: false
+            # the SAN of the certificate present inside the container
+            serverName: ""
+            # set insecureSkipVerify=true, if the certificate verification has to be skipped
+            insecureSkipVerify: false
+
+      ingress:
+        # For help configuring the ingress block including TLS, see the Helm chart documentation
+        # https://github.com/pegasystems/pega-helm-charts/blob/master/charts/pega/README.md#ingress
+
+        # Enter the domain name to access web nodes via a load balancer.
+        #  e.g. web.mypega.example.com
+        domain: "YOUR_WEB_NODE_DOMAIN"
+        tls:
+          # Enable TLS encryption
+          enabled: true
+          # secretName:
+          # useManagedCertificate: false
+          # ssl_annotation:
+
+      replicas: 1
+      javaOpts: ""
+      pegaDiagnosticUser: ""
+      pegaDiagnosticPassword: ""
+
+      deploymentStrategy:
+        rollingUpdate:
+          maxSurge: 1
+          maxUnavailable: 0
+        type: RollingUpdate
+
+      livenessProbe:
+        port: 8081
+
+      # Optionally overridde default resource specifications
+      # cpuRequest: 2
+      # memRequest: "12Gi"
+      # cpuLimit: 4
+      # memLimit: "12Gi"
+      # initialHeap: "4096m"
+      # maxHeap: "8192m"
+
+      # To configure an alternative user for custom image, set value for runAsUser.
+      # See, https://github.com/pegasystems/pega-helm-charts/blob/master/charts/pega/README.md#security-context
+      # securityContext:
+      #   runAsUser: 9001
+
+      hpa:
+        enabled: true
+
+      # Set enabled to true to include a Pod Disruption Budget for this tier.
+      # To enable this budget, specifiy either a pdb.minAvailable or pdb.maxUnavailable
+      # value and comment out the other parameter.
+      pdb:
+        enabled: false
+        minAvailable: 1
+        # maxUnavailable: "50%"
+
+    - name: "batch"
+      # Create a background tier for batch processing.  This tier uses
+      # a collection of background node types and will not be exposed to
+      # the load balancer.
+      nodeType: "BackgroundProcessing,Search,Batch,RealTime,Custom1,Custom2,Custom3,Custom4,Custom5,BIX"
+
+      replicas: 1
+      javaOpts: ""
+
+      pegaDiagnosticUser: ""
+      pegaDiagnosticPassword: ""
+
+      deploymentStrategy:
+        rollingUpdate:
+          maxSurge: 1
+          maxUnavailable: 0
+        type: RollingUpdate
+
```

Comment:
testing explicitly overridden liveness values that do not exceed the maximum of classloader retry timeout of 180s.